app/models/event.rb

```diff
@@ -1,12 +1,24 @@
 class Event < ApplicationRecord
   belongs_to :place 
   belongs_to :user
+<<<<<<< HEAD
+  has_and_belongs_to_many :members, class_name: "User", dependent: :destroy 
+=======
   has_and_belongs_to_many :members, class_name: "User"
   accepts_nested_attributes_for :place
+>>>>>>> 8061266ad2cad7e517e5965faec4fc6a01cc1909
 
   validates :name, name_lenght: { minimum: 5 }
 
   def has_member?(user)
     members.include?(user)
   end
+
+  def before_destroy
```

Comment:
In model you don't have access to flash message. This is something you can prepare on controller level and display on view. When you want to check if event has members it will be better to check that on validation level or for now in controller.